packages/notices/src/store/reducer.js

```diff
@@ -26,6 +26,9 @@ const notices = onSubKey( 'context' )( ( state = [], action ) => {
 
 		case 'REMOVE_NOTICES':
 			return state.filter( ( { id } ) => ! action.ids.includes( id ) );
+
+		case 'REMOVE_ALL_NOTICES':
+			return state.filter( ( { type } ) => type !== action.noticeType );
```

Comment:
This isn't considering the provided context in the action object. Should it?

Reply:
> This isn't considering the provided context in the action object. Should it?

The [`onSubKey`](https://github.com/wordpress/gutenberg/blob/945d99d31711e0c128db7609a4708c82d3aa55aa/packages/notices/src/store/reducer.js#L15) in the reducer takes care of that actually.

I have tests that cover removing notices from a specific context: https://github.com/wordpress/gutenberg/blob/2a5db119b8ed8e16002de157a5edf37ff47dffe6/packages/notices/src/store/test/reducer.js#L230